File: CRS.Web/CllientApp/src/app/shared/model/Customers/customer.ts
import { Vehicle } from '../Vehicles/vehicle';


export class Customer {
    id: number;
    typeOfCustomerID: number;
    dateTimeAddCustomer?: Date;
    companyName: string;
    name: string;
    surname: string;
    province: string;
    city: string;
    zipCode: string;
    street: string;
    numberOfBuilding: string;
    numberOfApartment: string;
    email: string;
    phone: string;
    vehicles: Vehicle[];
}

File: CRS.Data/Entities/Customer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CRS.Data.Entities
{
    public class Customer:BaseEntity
    {
        public int TypeOfCustomerID { get; set; }
        public DateTime DateTimeAddCustomer { get; set; }
        public string CompanyName { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Province { get; set; }
        public string ZipCode { get; set; }
        public string City { get; set; }
        public string Street { get; set; }
        public string NumberOfBuilding { get; set; }
        public string NumberOfApartment { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public virtual ICollection<Order> Orders { get; set; }
        public virtual TypeOfCustomer TypeOfCustomer { get; set; }
        public virtual ICollection<Vehicle> Vehicles { get; set; }
    }
}
